### PR TITLE
Add check answers page guidance

### PR DESCRIPTION
**Pull request by timpaul**
Add a new design pattern for 'Check your answers' pages

### CODE REVIEW

File: service-manual/user-centred-design/resources/patterns/check-your-answers-pages.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+
+
+# Research
+
+You can help by researching the following questions:
+
+- what effect does the wording, styling and position of the ‘Change’ links have on their usage?
+- for long, multi-section transactions, do we need one of these pages at the end of each section?
+
+[Update the Hackpad with your findings]()
```
Comment (quis) [inline service-manual/user-centred-design/resources/patterns/check-your-answers-pages.md]: Missing link?

Comment (timpaul) [inline service-manual/user-centred-design/resources/patterns/check-your-answers-pages.md]: Ta for spotting that

**Comment by jabley** [comment]
Minor content issues with dashes versus em-dashes.

Needs corresponding redirects for the renamed resource. I'm not sure where we put them these days. I'll check and update this pull request.

**Comment by alexmuller** [comment]
> Needs corresponding redirects for the renamed resource. I'm not sure where we put them these days.

https://github.gds/gds/router-data/blob/master/data/slug-changes.csv

**Comment by daibach** [comment]
@timpaul are you happy for this to be merged?

I've submitted a pull request on router-data for the redirect. That should be deployed at the same time as this.

**Comment by timpaul** [comment]
Yup - very happy for that to be merged

**Pull request merged**